sympy/physics/quantum/spin.py

```diff
@@ -856,39 +856,42 @@ def _eval_wignerd(self):
         alpha = sympify(self.alpha)
         beta = sympify(self.beta)
         gamma = sympify(self.gamma)
-        if not j.is_number:
-            raise ValueError(
-                'j parameter must be numerical to evaluate, got %s' % j)
-        r = 0
-        if beta == pi/2:
-            # Varshalovich Equation (5), Section 4.16, page 113, setting
-            # alpha=gamma=0.
-            for k in range(2*j + 1):
-                if k > j + mp or k > j - m or k < mp - m:
-                    continue
-                r += (S.NegativeOne)**k*binomial(j + mp, k)*binomial(j - mp, k + m - mp)
-            r *= (S.NegativeOne)**(m - mp) / 2**j*sqrt(factorial(j + m) *
-                    factorial(j - m) / (factorial(j + mp)*factorial(j - mp)))
+        if alpha==0 and beta==0 and gamma==0:
+            return KroneckerDelta(m,mp)
         else:
```

Comment:
It isn't necessary to indent everything below the `if` here (the else clause isn't needed).

Reply:
Also use space after comma and either side of `==`

Reply:
Hey @oscarbenjamin !  Worked on the comments and updated the PR!
Thanks for the comments!